src/components/unique-token/UniqueTokenImage.js

```diff
@@ -102,12 +103,13 @@ const UniqueTokenImage = ({
               onError={handleError}
               onLoad={onLoad}
               resizeMode={ImgixImage.resizeMode[resizeMode]}
+              size={size}
               source={{ uri: image }}
               style={position.coverAsObject}
             />
             {!loadedImg && lowResUrl && (
               <ImageTile
-                {...(isCard && { fm: 'png' })}
+                fm="png"
```

Comment:
why was it?

Reply:
I wanted to reduced the amount of cached images as much as possible, so the idea was to keep the same formatting/sizing in as many places as possible. in this case we can use the same loading image for the card and the expanded state image